src/main/resources/default/taglib/t/deleteButton.html.pasta

```diff
@@ -1,11 +1,12 @@
 <i:arg type="String" name="url"/>
 <i:arg type="Page" name="page" default=""/>
 <i:arg name="class" type="String" default="" description="Lists additional CSS classes to apply to the field."/>
+<i:arg name="withLabel" type="boolean" default="false" description="Specifies if the button should have an appropriate label."/>
```

Comment:
just `label` ?

Reply:
I don't like a boolean flag indicating whether to use a certain label. I wonder whether the usual mechanism using an empty string (or an empty) key would be better